geomesa-kafka/geomesa-kafka-datastore/README.md

```diff
@@ -0,0 +1,51 @@
+#GeoMesa KafkaDataStore
+The `KafkaDataStore` is an implementation of the geotools `DataStore` interface that is backed by a Kafka message broker.
+The implementation supports the ability for feature producers to persist data into the data store and for consumers to 
+pull data from the datastore.  The data store supports both live and replay consumer modes.  Consumers operating in 
+live mode pull data from the end of the message queue (e.g latest time) while consumers operating in replay mode pull data 
+from a specified time interval in the past.  
+
+##Usage/Configuration
+Clients of the `KafkaDataStore` select between live mode and replay mode through hints provided by the SimpleFeatureType
+object passed to `createSchema(SimpleFeatureType)`.  These hints are added through the use of the following helper
+methods:
+    
+    
+    KafkaDataStoreHelper.prepareForLive(sft: SimpleFeatureType, zkPath: String) : SimpleFeatureType
+    KafkaDataStoreHelper.prepareForReplay(sft: SimpleFeatureType, rc: ReplayConfig) : SimpleFeatureType
```

Comment:
Could we rename these to something like 'configureForReplay' - or if the original SFT is not modified, something like 'createReplaySft' ?